### PR TITLE
webpage: link updates

### PR DESCRIPTION
**Pull request by tooomm**
follow up to https://github.com/mtgjson/mtgjson/pull/393

### CODE REVIEW

File: web/index.pug [hunk 1]
```diff
@@ -173,26 +173,26 @@ block contents
 						li
 							a(href="https://github.com/fenhl",target="_blank") Fenhl
 				li
-					a(href="http://wiki.mtgsalvation.com/article/Main_Page",target="_blank") MTG Salvation Wiki
+					a(href="https://wiki.mtgsalvation.com/article/Main_Page",target="_blank") MTG Salvation Wiki
```
Comment (fenhl) [inline web/index.pug]: The current URL is <https://mtg.gamepedia.com/>.

Comment (tooomm) [inline web/index.pug]: You are so right, i missed the redirect

**Comment by ZeldaZach** [comment]
Add a changelog entry and i'll merge this in.

**Comment by tooomm** [comment]
This change is not related to mtgjson code or output, i suggest we don't spam the changelog with little adjustments/fixes to the website.

If we want a changelog for everything, the website should have it's own repo with it's own changelog.
A clean mtgjson repo is always not bad, the website code doesn't really fit anyway!
By the way, the webpage could be hosted right off github pages...

**Pull request merged**